packages/api/src/valueObjects/Thaiti.ts

```diff
@@ -0,0 +1,21 @@
+export default class Thaiti {
+    private thaiti: string;
+
+    constructor(thaiti: string) {
+        if (!Thaiti.isThaiti(thaiti)) {
+            throw new Error("Invalid Thaiti: " + thaiti);
+        }
+
+        this.thaiti = thaiti;
+    }
+
+    static isThaiti(thaiti: string | undefined): boolean {
+        const regex1 = /^[A-Z][0-9]{5}$/i; // Regex : 1 lettre + 5 chiffres
+        const regex2 = /^\d{6}$/; // Regex : 6 chiffres
+        return typeof thaiti === "string" && (regex1.test(thaiti) || regex2.test(thaiti));
```

Comment:
```suggestion
        const regex = /^[A-Z0-9][0-9]{5}$/i; // Regex : 1 lettre + 5 chiffres
        return typeof thaiti === "string" && (regex.test(thaiti));
```

idem pour synthétiser, et pourrait être fait pour les ThaitiT

Reply:
Et autre question sur ces regex : ne risque-t-on pas de croiser des thaiti(T) où la lettre serait en minuscule ? Ton code ne l'accepterait pas

Reply:
@alice-telescoop le i à la fin de ma REGEX rends la REGEX insensible à la casse. J'ai même ajouté ça dans le test unitaire pour valider que tout est ok en cas de lettre minuscule.